profiler-egg-v4/REPORT.md

```
@@ -0,0 +1,382 @@
+| isolate-47316 | 9008 | 54.8% | 22.2% | 5.0% | 23.0% |
+| isolate-47317 | 9021 | 53.2% | 0.0% | 5.9% | 46.8% |
+| isolate-47318 | 9016 | 54.1% | 0.0% | 5.0% | 45.7% |
+| isolate-47319 | 9031 | 53.7% | 0.0% | 5.5% | 46.3% |
+
+---
+
+## Top Hotspot Functions (Aggregated)
+
+### 1. Promise/Async Hook Functions (Critical - ~15-18% combined)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)